Jenkinsfile.nightly

```diff
@@ -28,23 +27,27 @@ pipeline {
-                    def repositoryListPath = "./script/repository-list.txt"
-                    def projectVariableMap = ['kiegroup_droolsjbpm-build-bootstrap': "kieVersion", 'jboss-integration_izpack': "izpackVersion", 'jboss-integration_installer-commons': "installerCommonsVersion"]
-                    def nightlyStages = load("$WORKSPACE/nightly.stages")
+                        // This is the map project, variable to store the version from this project
+                        def repositoryListPath = "./script/repository-list.txt"
+                        def projectVariableMap = ['kiegroup_droolsjbpm-build-bootstrap': "kieVersion", 'jboss-integration_izpack': "izpackVersion", 'jboss-integration_installer-commons': "installerCommonsVersion"]
+                        def nightlyStages = load("$WORKSPACE/nightly.stages")
 
-                    configFileProvider([configFile(fileId: "49737697-ebd6-4396-9c22-11f7714808eb", variable: 'PRODUCTION_PROJECT_LIST')]) {
-                        println "Reading file ${repositoryListPath} and merging it with ${PRODUCTION_PROJECT_LIST} jenkins file"
-                        def communityProjectListFile = readFile repositoryListPath
-                        def projectCollection = communityProjectListFile.readLines()
-                        def productionProjectListFile = readFile "${env.PRODUCTION_PROJECT_LIST}"
-                        projectCollection.addAll(productionProjectListFile.readLines())
-                        projectCollection.removeAll { 'droolsjbpm-tools'.equals(it.toLowerCase()) } // The droolsjbpm-tools is skiped
-                        println "File ${repositoryListPath} and ${env.PRODUCTION_PROJECT_LIST} jenkins file merged in ${projectCollection}"
-                        nightlyStages(SETTINGS_XML_ID, projectCollection, "rhba/nightly", projectVariableMap)
-                    }
+                        configFileProvider([configFile(fileId: "49737697-ebd6-4396-9c22-11f7714808eb", variable: 'PRODUCTION_PROJECT_LIST')]) {
+                            println "Reading file ${repositoryListPath} and merging it with ${PRODUCTION_PROJECT_LIST} jenkins file"
+                            def communityProjectListFile = readFile repositoryListPath
+                            def projectCollection = communityProjectListFile.readLines()
+                            def productionProjectListFile = readFile "${env.PRODUCTION_PROJECT_LIST}"
+                            projectCollection.addAll(productionProjectListFile.readLines())
+                            projectCollection.removeAll { 'droolsjbpm-tools'.equals(it.toLowerCase()) } // The droolsjbpm-tools is skiped
+                            println "File ${repositoryListPath} and ${env.PRODUCTION_PROJECT_LIST} jenkins file merged in ${projectCollection}"
+                            nightlyStages(SETTINGS_XML_ID, projectCollection, "rhba/nightly", projectVariableMap)
+                        }
+                    }, 2, 720*60)
```

Comment:
what is the reasoning behind this timeout 720 minutes aka 12 hours reattempt ?

Reply:
@mareknovotny 720 was the original value, according to the latest executions it's between 4 and 6,5 hours, so we can decrease it to 8 hours?